tests/input/html/test_html.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+import os
+import shutil
+import typing
+
+import pytest
+
+from preview_generator.manager import PreviewManager
+from preview_generator.utils import executable_is_available
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+CACHE_DIR = "/tmp/preview-generator-tests/cache"
+IMAGE_FILE_PATH = os.path.join(CURRENT_DIR, "test_html.html")
+IMAGE_FILE_PATH_NO_EXTENSION = os.path.join(CURRENT_DIR, "html_no_extension")  # nopep8
```

Comment:
I suggest you to rename `IMAGE_FILE_PATH` to   `HTML_FILE_PATH` 
and `IMAGE_FILE_PATH_NO_EXTENSION` to `HTML_FILE_PATH_NO_EXTENSION`
to be more explicit (it's a naming mistake that already exist in some of current test file).